requirements/prod.txt

```diff
@@ -1,22 +0,0 @@
-# Production requirements - the absolute minimum to be able to run the code
```

Comment:
Make sure you save a `requirements.txt` or a `environment.yml` file for reproducibility of your code. This is very important, both if you want to get back to some code in this project in the future and if you were to share it with others in a reproducible research setting.

You can see how this can be done in [conda here](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#sharing-an-environment)